packaging/hudi-hadoop-mr-bundle/pom.xml

```diff
@@ -110,6 +135,74 @@
                   <pattern>com.google.common.</pattern>
                   <shadedPattern>org.apache.hudi.com.google.common.</shadedPattern>
                 </relocation>
+                <!-- The classes below in org.apache.hadoop.metrics2 package come from
```

Comment:
I assume all of this is repeated? does maven offer a way to reuse the include? I am wondering if we can build a `hudi-hbase-shaded` package and simply include that everywhere. will be easier to maintain?

Reply:
+1

Reply:
Yes, we can have `hudi-hbase-shaded` as a separate module.  I'll see if this can be done soon; if not, I'll take it as a follow-up.

Reply:
HUDI-3673 is created as a follow-up.